### ISSUE TITLE
TypeScript build error TS1064

### ISSUE THREAD

**Issue by mdorda** [glslify/rollup-plugin-glslify#19]
When package is installed, TypeScript build fails on:

```
node_modules/rollup-plugin-glslify/index.d.ts(11,1): error TS1046: Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.
```

It can be "fixed" using `skipLibCheck: true`, but it also disables validation of all `*.d.ts` local files.

**Comment by Lutymane** [comment]
Thanks for the issue!

But the proper solution will be to have `declare const` instead of introducing a ghost export that doesn't actually exist

**Comment by mdorda** [comment]
Thanks! You are right, I see you open PR with proper fix. I closed my PR in favor of https://github.com/glslify/rollup-plugin-glslify/pull/21

**Comment by pschroen** [comment]
Published rollup-plugin-glslify@1.3.1. If you can confirm the TypeScript build is working now. Thanks!

**Comment by mdorda** [comment]
Works like a charm, thanks!
